ironmon_tracker/Options.lua

```diff
@@ -53,49 +54,58 @@ Options = {
 	},
 }
 
+-- The order of these iconsets cannot change, as user Settings reference them by their index number
 Options.IconSetMap = {
-	totalCount = 5,
-	["1"] = {
+	{
 		name = "Original", -- The name of the icon set which is displayed on the Tracker Setup screen
+		author = "Besteon", -- The name of the creator of the icon set which is displayed on the Tracker Setup screen
 		folder = "pokemon", -- The folder within the tracker files where each icon is stored, expected to be in /ironmon-tracker/images/
 		extension = ".gif", -- The file extension for each icon, expected that all icons use the same file extension
 		yOffset = 0, -- A number of pixels to shift the drawing of the icon downward
 		adjustQuestionMark = true, -- If true, will shift the question mark icon on the RouteInfo screen downward `yOffset` pixels
-		author = "Besteon", -- The name of the creator of the icon set which is displayed on the Tracker Setup screen
 	},
-	["2"] = {
+	{
 		name = "Stadium",
+		author = "AmberCyprian",
 		folder = "pokemonStadium",
 		extension = ".png",
 		yOffset = 4,
-		adjustQuestionMark = false,
-		author = "AmberCyprian",
 	},
-	["3"] = {
+	{
 		name = "Gen 7+",
+		author = "kittenchilly",
 		folder = "pokemonUpdated",
 		extension = ".png",
 		yOffset = 2,
 		adjustQuestionMark = true,
-		author = "kittenchilly",
 	},
-	["4"] = {
+	{
 		name = "Explorers",
+		author = "Fellshadow",
 		folder = "pokemonMysteryDungeon",
 		extension = ".png",
 		yOffset = 4,
-		adjustQuestionMark = false,
-		author = "Fellshadow",
 	},
-	["5"] = {
+	{
 		name = "Virtual Pet",
+		author = "Ryastoise",
 		folder = "pokemonVPet",
 		extension = ".png",
 		yOffset = 2,
-		adjustQuestionMark = false,
-		author = "Ryastoise",
+	},
+	{
+		name = "Walking Pals",
+		author = "UTDZac",
+		folder = "spritesWalkingPals",
+		extension = ".png",
+		isAnimated = true,
+		iconKey = "WalkingPals",
 	},
```

Comment:
Good note. I'll add in a `source` property for the ones that I can find. I'm going to comment-out this property (save space), since we don't have a use for it yet. I imagine it will be easy to add in some external link to open up the source later; at which time we can uncomment these.